### PR TITLE
Update instruction

### PR DESCRIPTION
**Pull request by Abhishek01039**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ Built with clean architecture and domain driven design with clear seperation of
 
 ## 💡 Get started
 
+1. Download the configuration file from the Firebase Console (the file is called google-services.json). Add this file into the android/app directory within your Flutter project.
+
+2. Download the configuration file from the Firebase Console (the file is called GoogleService-Info.plist). Add this file via [this](https://firebase.flutter.dev/docs/installation/ios#installing-your-firebase-configuration-file) instruction.
+
+3. Create file (the file is called firebase_config.js). This file should contains the configuration for firebase as you can see `firebase_config.js.example`.
```
Comment (chethanprabhakar) [inline README.md:37]: 1. Can you add the relative path `web/script/firebase_config.js.example`?
2. Consider renaming the folder `web/script/` as `web/config/`
3. Please add config code snippet from `firebase_config.js.example` in the README

**Review by chethanprabhakar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Abhishek01039) [reply README.md:37]: Sure

File: README.md [hunk 2]
```diff
@@ -30,6 +30,34 @@ Built with clean architecture and domain driven design with clear seperation of
 
 ## 💡 Get started
 
+1. Download the configuration file from the Firebase Console (the file is called google-services.json). Add this file into the android/app directory within your Flutter project.
+
+2. Download the configuration file from the Firebase Console (the file is called GoogleService-Info.plist). Add this file via [this](https://firebase.flutter.dev/docs/installation/ios#installing-your-firebase-configuration-file) instruction.
+
+3. Create file (the file is called firebase_config.js). This file should contains the configuration for firebase as you can see `web/config/firebase_config.js.example`.
```
Comment (chethanprabhakar) [inline README.md:37]: Screenshot 2021-08-04 at 1 23 22 PM

Please add relative url `web/config/firebase_config.js`

**Review by chethanprabhakar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Abhishek01039) [reply README.md:37]: Yeah sure

**Review by chethanprabhakar — APPROVED** [review]

**Pull request merged**